_storage/amazon-s3-deploy.md

```diff
@@ -0,0 +1,478 @@
+---
+layout: storage
+permalink: /:collection/:path.html
+---
+# Configure a hub on Amazon EC2
+{:.no_toc}
+
+This teaches you how to run a Gaia hub on Amazon EC2. Amazon EC2 is an affordable and convenient cloud computing provider. This example uses Amazon EC2 together with an EB3 instance for file storage.
+
+* TOC
+{:toc}
+
+<div class="uk-card uk-card-default uk-card-body">
+<h5>Is this tutorial for you?</h5>
+
+<p>This documentation is appropriate for advanced power users who are familiar with command line tools, <code>ssh</code>, and basic editing configuration files.</p>
+
+<p>If you are planning on running an <emphasis>open-membership hub</emphasis> or an <emphasis>application-specific hub</emphasis>, see <a href="hub-operation.html">the section on Hub Operation</a>.</p>
+
+</div>
+
+## Prerequisites you need
+
+This procedure uses Amazon AWS to choose and configure an Amazon Machine Image
+(AMI) running a Gaia service. For this reason, you should have an AWS account
+on the <a href="https://aws.amazon.com/free/" target="\_blank">Amazon AWS free
+tier</a>, personal account, or corporate account.  These instructions assume you
+are using a free tier account.
+
+These instructions assume you have already created a free <a
+href="https://www.freenom.com" target="\_blank">domain through the freenom
+service</a>. If you have another domain, you can use that instead.
+
+Finally, setting up the SSL certificates on your EC2 instance requires you to use the terminal command line on your workstation. Make sure you have the `watch` command installed using the `which` command.
+
+```
+$ which watch
+/usr/local/bin/watch
+```
+
+If `watch` is not located, install it on your workstation.
+
+## Task 1: Launch an EC2 instance
+
+1. Visit the <a href="https://aws.amazon.com/free/" target="\_blank">AWS Free Tier page</a> and choose **Sign in to the Console**.
+
+   ![](/storage/images/aws-console.png)
+
+2. Make sure your region is set to one close to layout.
+
+   ![](/storage/images/us-west-2.png)
+
+3. Under  **Build a solution** choose **Launch a virtual machine**.
+
+   The system opens the EC2 dashboard.
+
+4. Enter `Blockstack Gaia` in the search bar.
+
+   The system finds AMIs in the Marketplace and the Community.
+
+5. Choose **Community AMIs**.
+
+   The system displays the available Gaia hub images.
+
+   ![](/storage/images/gaia-community.png)
+
+6. Select the most recent version of the image.
```

Comment:
I'm finding it's hard to spot which AMI is the most recent by glancing – do I have to drill in somewhere to find that out?

<img width="1143" alt="screen shot 2019-02-20 at 15 24 17" src="https://user-images.githubusercontent.com/28991/53098355-b945d600-3523-11e9-87ae-9bb98792f1fc.png">

Reply:
Or is the most recent one highest up in the list? If so, we should probably indicate that.

Reply:
@markmhx Brought this up in the meeting yesterday. @wileyj is going to clean up the list; when he does, I'll retake the capture.  Then, there will be two images for each release. The images will have the same version but differ in the type of storage they provide.  The current version is 1.0.1 and user can choose ephemeral or ebs storage. I'll clarify this further in the docs.  I'll explain the version format in the docs. I think we can release with this capture and the explanation I just added.